packages/electrode-archetype-react-app-dev/config/babel/babelrc-server

```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "es2015-node"
```

Comment:
so far we usually put React code in client and load them from server.  for the most part, the server code don't expect to have React code.  app's `server/.babelrc` doesn't have to extend this one anyways.